src/poprox_recommender/test_offline.py

```diff
@@ -89,25 +89,38 @@ def recsys_metric(mind_data: MindData, request: RecommendationRequest, recommend
                 request.num_recs,
             )
         try:
-            recommendations = pipeline(
-                {
-                    "candidate": ArticleSet(articles=request.todays_articles),
-                    "clicked": ArticleSet(articles=request.past_articles),
-                    "profile": request.interest_profile,
-                }
-            )
+            if request.interest_profile.click_history.article_ids:
+                recommendations = pipeline(
+                    {
+                        "candidate": ArticleSet(articles=request.todays_articles),
+                        "clicked": ArticleSet(articles=request.past_articles),
+                        "profile": request.interest_profile,
+                    }
+                )
+                personalized = 1
+            else:
+                recommendations = fallback(
+                    {
+                        "candidate": ArticleSet(articles=request.todays_articles),
+                        "clicked": ArticleSet(articles=request.past_articles),
+                        "profile": request.interest_profile,
+                    }
+                )
+                personalized = 0
```

Comment:
We should abstract this better (in-pipeline fallback support?) but for now this gets the eval working. Also record which pipeline was used for each user.